aviary/variable_info/variable_meta_data.py

```diff
@@ -2099,6 +2099,18 @@
     default_value=0.0,
 )
 
+add_meta_data(
+    Aircraft.Engine.SLS_SHAFT_POWER_MAX,
+    meta_data=_MetaData,
+    historical_name={"GASP": 'INPROP.HPMSLS',
+                     "FLOPS": None,
+                     "LEAPS1": None
+                     },
+    units='hp',
+    desc='maximum shaft power of a turboprop',
```

Comment:
```suggestion
    desc='maximum shaft power of an engine',
```
anything from a turboshaft, recip engine, to even a black-box motor model can use this variable